metals/src/main/scala/scala/meta/internal/builds/ShellRunner.scala

```diff
@@ -70,12 +56,7 @@ class ShellRunner(
     val classpath = Fetch
       .create()
       .withDependencies(dependency)
-      .withRepositories(
-        Repository.ivy2Local(),
-        Repository.central(),
-        mavenLocal,
-        sonatypePublic,
-      )
+      .withRepositories(ShellRunner.defaultRepositories: _*)
```

Comment:
wait, does this override `COURSIER_REPOSITORIES`?

Reply:
I don't think it does, COURSIER_REPOSITORIES have the priority as far as I remember :thinking:

Reply:
ok